doc/source/user_guide/api_examples.rst

```diff
@@ -0,0 +1,152 @@
+
+    client = Client("http://localhost:9997")
+    # The chatglm2 model has the capabilities of "chat" and "embed".
+    model_uid = client.launch_model(model_name="chatglm2", 
+                                    model_format="ggmlv3",
+                                    model_size_in_billions=6,
+                                    quantization="q4_0")
+    model = client.get_model(model_uid)
+
+    chat_history = []
+    prompt = "What is the largest animal?"
+    # If the model has "generate" capability, then you can call
+    # model.generate API.
+    model.chat(
+        prompt,
+        chat_history,
+        generate_config={"max_tokens": 1024}
+    )
+
+
+Embedding
+~~~~~~~~~
+
+List available built-in embedding models:
+
+.. code-block::
+
+    >>> xinference registrations -t embedding
+
+    Type       Name                     Language      Dimensions  Is-built-in
+    ---------  -----------------------  ----------  ------------  -------------
+    embedding  bge-base-en              ['en']               768  True
+    embedding  bge-base-en-v1.5         ['en']               768  True
+    embedding  bge-base-zh              ['zh']               768  True
+    ...
+
+
+Launch an embedding model and embed a text:
+
+.. code-block::
+
+    from xinference.client import Client
+
+    client = Client("http://localhost:9997")
+    # The bge-small-en-v1.5 is an embedding model, so the `model_type` needs to be specified.
+    model_uid = client.launch_model(model_name="bge-small-en-v1.5", model_type="embedding")
+    model = client.get_model(model_uid)
+
+    input_text = "what is the capital of China?"
+    model.create_embedding(input_text)
+
+Output
+
+.. code-block::
+
+    {'object': 'list',
+     'model': 'da2a511c-6ccc-11ee-ad07-22c9969c1611-1-0',
+     'data': [{'index': 0,
+     'object': 'embedding',
+     'embedding': [-0.014207549393177032,
+        -0.01832585781812668,
+        0.010556723922491074,
+        ...
+        -0.021243810653686523,
+        -0.03009396605193615,
+        0.05420297756791115]}],
+     'usage': {'prompt_tokens': 37, 'total_tokens': 37}}
+
+Image
+~~~~~
+
+List available built-in image models:
+
+.. code-block::
+
+    >>> xinference registrations -t image
+
+    Type    Name                          Family            Is-built-in
+    ------  ----------------------------  ----------------  -------------
+    image   stable-diffusion-v1-5         stable_diffusion  True
+    image   stable-diffusion-xl-base-1.0  stable_diffusion  True
+
+
+Launch an image model and generate an image by prompt:
+
+.. code-block::
+    
+    from xinference.client import Client
+
+    client = Client("http://localhost:9997")
+    # The stable-diffusion-v1-5 is an image model, so the `model_type` needs to be specified.
+    # Additional kwargs can be passed to AutoPipelineForText2Image.from_pretrained here.
+    model_uid = client.launch_model(model_name="stable-diffusion-v1-5", model_type="image")
```

Comment:
It's a common practice to use a dot in versioning model names, as seen in many LLM names like vicuna-v1.3, mistral-v0.1.

The name stable-diffusion-v1-5 seems a bit counterintuitive in this context. I would suggest renaming it to stable-diffusion-v1.5 to maintain consistency and intuitiveness in the naming convention.

Reply:
OK. I will fix it. Thanks.